app/controllers/journals_controller.rb

```diff
@@ -37,7 +37,7 @@ def update
 
   def destroy
     @journal.destroy
-    redirect_to @journaled, make_notice
+    redirect_to request.referer, make_notice
```

Comment:
`redirect_to :back` does the same, with this caveat:

> When using redirect_to :back, if there is no referrer, ActionController::RedirectBackError will be raised. You may specify some fallback behavior for this case by rescuing ActionController::RedirectBackError.

Referer might be empty for various reasons (e.g. privacy extensions), it would probably be sufficient to catch `ActionController::RedirectBackError` in `ApplicationController` and redirect to the root as a fallback.

Reply:
I used that first, and had exactly that error.